containers/datalab/content/run.sh

```diff
@@ -256,12 +96,6 @@ if [ -n "${vm_project}" ] && [ "${vm_project}" != "no-project-id" ]; then
    export VM_ZONE=$(curl -s "${compute_metadata_url}/instance/zone" -H "Metadata-Flavor: Google" | sed 's/.*zones\///')
 fi
 
-# Install the kernel gateway server extension, if a kernel gateway URL has been specified
-if [ -n "${KG_URL}" ]
```

Comment:
We have more references in `jupyter.ts` that uses `KG_URL` to set up the server to connect to the gateway.